blocks/article-tag-block/_index.scss

```diff
@@ -0,0 +1,10 @@
+@use "@wpmedia/arc-themes-components/scss";
+
+.b-article-tag {
+	&__item {
```

Comment:
We don't need this class as it is applied to the `Pill` component which can be styled via the component overrides

Reply:
updated and removed